docs/references.bib

```diff
@@ -1,3 +1,10 @@
+@article{pooladian2021entropic,
```

Comment:
The citation was already included in the `references.bib` as `pooladian:21`, please remove this one use the above-mentioned.